src/utils/geomFunctions.js

```diff
@@ -20,3 +20,24 @@ export function getPolygonCenter (polygonBounds) {
   let yMean = (polygonBounds.yMin + polygonBounds.yMax) / 2;
   return {xMean: xMean, yMean: yMean};
 }
+
+export function createBbox (xMean, xDiviation, yMean, yDiviation) {
+  return [
+    xMean - xDiviation,
+    yMean - yDiviation,
+    xMean + xDiviation,
+    yMean + yDiviation
+  ];
+}
+
+export function arraysOfCoordsEqual(arr1, arr2) {
+  if(arr1.length !== arr2.length)
+    return false;
+  for(var i = arr1.length; i--;) {
+    for(var j = arr1.length; j--;) {
```

Comment:
This doesn't look correct, needs to check for `arr1[i].length`. And before the second loop, a `if (arr1[i].length !== arr2[i].length) return false;`.